libres/old_tests/enkf/test_enkf_export_inactive_cells.cpp

```diff
@@ -31,8 +31,8 @@ namespace enkf_main {
 void init_active_runs(const res_config_type *res_config,
                       const ert_run_context_type *run_context);
 
-void reset_run_path(runpath_list_type *runpath_list,
-                    const ert_run_context_type *run_context);
+std::vector<Runpath>
+run_context_get_runpath_list(const ert_run_context_type *run_context);
```

Comment:
Same comment about `runpath_list` -> `runpaths`.

Reply:
Changed it